components/Avatar/Avatar.unit.test.jsx

```diff
@@ -0,0 +1,33 @@
+
+describe('<Avatar />', () => {
+  describe('Should match snapshot', () => {
+    it('With picture', () => {
+      expect(
+        renderer.create(<Avatar picture={encodedPicture} />).toJSON(),
+      ).toMatchSnapshot();
+    });
+
+    it('Without picture', () => {
+      expect(renderer.create(<Avatar />).toJSON()).toMatchSnapshot();
+    });
+
+    it('With notification', () => {
+      expect(
+        renderer.create(<Avatar hasNotification />).toJSON(),
+      ).toMatchSnapshot();
+    });
+
+    it('With text', () => {
+      expect(
+        renderer.create(<Avatar text="Profile" />).toJSON(),
+      ).toMatchSnapshot();
+    });
+  });
```

Comment:
what do you think to create a case using all props in the component?

Reply:
is it better?

Reply:
yes. I believe that you keep "safe" the all situations in use this component.

Reply:
I thought you meant I should replace all

Reply:
done